db/schema.rb

```diff
@@ -24,6 +24,21 @@
     t.string "ProductVersion", limit: 32, null: false
   end
 
+  create_table "access_request", id: :serial, force: :cascade do |t|
```

Comment:
We have a migration called `DeleteAccessRequestTable` that is creating the table 🤔